README.md

```diff
@@ -34,22 +34,22 @@ Since we combined a large number of source libraries, we have done some work to
 ```bash
 git status # make sure you have a clean working tree, in case you want to revert
 
-npx elm-review --template elmcraft/core-extra/upgrade#2.0.0 --fix
-# You can run this command multiple times, if you want to abort and fix stuff automatically
+npx elm-review --template elmcraft/core-extra/upgrade/2.0.0 --fix
```

Comment:
The reason I pinned this to a tag is so that we don't have to maintain this migration script in our master branch forever. Future versions of core-extra could just have: "Upgrading from these legacy libraries? Follow the instructions here [link], then continue with our upgrade script."

or some such.


Maybe that doesn't matter so much? Or another option would be to have it point to a branch, which might be the best of both worlds: it can stay out of master, but be updated if we need to?

Reply:
I think having migration stuff as part of master forever is not a bad thing.
There's now one obvious place for all the migrations, including ones in the future and it's easier to keep all of them maintained at the same time (e.g. upgrading dependencies is just one-ish step)

To me it's the same reasoning as to why you keep old change logs around on master forever.
In any case, I don't feel strongly about this, so feel free to create a new branch from the state after this PR and delete it in master.

Reply:
Yeah makes sense. Let's do that then.